source/renderer/app/stores/AppStore.js

```diff
@@ -74,6 +79,22 @@ export default class AppStore extends Store {
     this.isAboutDialogOpen = false;
   };
 
+  @action _toggleAboutDialog = () => {
+    this.isAboutDialogOpen = !this.isAboutDialogOpen;
+  };
+
+  @action _openNetworkStatusDialog = () => {
+    this.isNetworkStatusDialogOpen = true;
+  };
+
+  @action _closeNetworkStatusDialog = () => {
+    this.isNetworkStatusDialogOpen = false;
+  };
+
+  @action _toggleNetworkStatusDialog = () => {
+    this.isNetworkStatusDialogOpen = !this.isNetworkStatusDialogOpen;
+  };
```

Comment:
This code looks like it would be possible to open both about and network status dialog at the same time?